k8s/namespaces/ia/ia-case-api/demo.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-case-api
+spec:
+  values:
+    java:
+      environment:
+        IA_FEEPAYMENT_ENABLED: "true"
+        IA_HOME_OFFICE_INTEGRATION_ENABLED: "true"
+        EM_BUNDLER_URL: "http://em-ccdorc-demo.service.core-compute-demo.internal"
```

Comment:
is there a reason this is here? and not in the helm chart? (in progress maybe?)

Reply:
This feature was moved to documents-api, let me clean up with follow up PR